tasks/connectors/github_dependabot/github_dependabot.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+require "graphql/client"
+require "graphql/client/http"
+module Kenna
+  module Toolkit
+    class GithubDependabot < Kenna::Toolkit::BaseTask
+      def self.metadata
+        {
+          id: "github_dependabot",
+          name: "github_dependabot Vulnerabilities",
+          description: "Pulls assets and vulnerabilitiies from github_dependabot",
+          options: [
+            { name: "github_access_token",
+              type: "api_key",
+              required: true,
+              default: nil,
+              description: "Github Access Token" },
+            { name: "kenna_api_key",
+              type: "api_key",
+              required: false,
+              default: nil,
+              description: "Kenna API Key" },
+            { name: "kenna_api_host",
+              type: "hostname",
+              required: false,
+              default: "api.kennasecurity.com",
+              description: "Kenna API Hostname" },
+            { name: "kenna_connector_id",
+              type: "integer",
+              required: false,
+              default: nil,
+              description: "If set, we'll try to upload to this connector" },
+            { name: "output_directory",
+              type: "filename",
+              required: false,
+              default: "output/github_dependabot",
+              description: "If set, will write a file upon completion. Path is relative to #{$basedir}" }
+          ]
+        }
+      end
+
+      # Github graphql configuration
+
+      http = GraphQL::Client::HTTP.new("https://api.github.com/graphql") do
+        def headers(_context)
+          { "Authorization": "Bearer #{GITHUB_DEPENDABOT_TOKEN}" }
+        end
+      end
+
+      Schema = GraphQL::Client.load_schema(http)
+
+      Client = GraphQL::Client.new(schema: Schema, execute: http)
+
+      # Graphql query
+
+      SECURITY_ADVISORY_QUERY = Client.parse <<-'GRAPHQL'
+        query {
+          organization(login: "KennaSecurity") {
```

Comment:
"KennaSecurity" should be modified to be a variable that can be passed through on the command line. As written, all runs only produce results for the KennaSecurity organization and assets are empty when the personal access token does not have permissions for KennaSecurity Dependabot alerts